about.html

```diff
@@ -0,0 +1,230 @@
+          all over the world.
+        </h1>
+        <p class="aboutTextBox">
+          For renewable energy technology to be a global public good - meaning available to all, and not just to 
+          the wealthy - it will be essential to remove roadblocks to knowledge sharing and technological transfer, 
+          including intellectual property rights barriers.
+          <br>
+          Essential technologies such as battery storage systems allow energy from renewables, 
+          like solar and wind, to be stored and released when people, communities and businesses need power.
+          A robust supply of renewable energy components and raw materials is essential. More widespread access 
+          to all the key components and materials - from the minerals needed to produce wind turbines and electricity 
+          networks, to electric vehicles - will be key.
+          <br>
+          It will take significant international coordination to expand and diversify manufacturing capacity globally. 
+          Moreover, greater investments are needed to ensure a just transition - including in people’s skills training, 
+          research and innovation, and incentives to build supply chains through sustainable practices that protect 
+          ecosystems and cultures.
+        </p>
+        <p class="locale">
+          <i
+            >Please contact us by Email for any further questions about the Green-energy
+             2022</i
+          >
+        </p>
+        <p class="mail">views.today@greenenergy.org</p>
+      </div>
+    </section>
+
+    <section class="commons" id="programs">
+      <h2>Shift energy subsidies from fossil fuels to renewable energy</h2>
+      <hr />
+
+      <p class="see_more_text">
+        Fossil-fuel subsidies are one of the biggest financial barriers hampering the world’s 
+        shift to renewable energy. The International Monetary Fund (IMF) says that about $5.9 trillion 
+        was spent on subsidizing the fossil fuel industry in 2020 alone, including through explicit subsidies, 
+        tax breaks, and health and environmental damages that were not priced into the cost of fossil fuels. 
+        That’s roughly $11 billion a day.
+      </p>
+      <img
+        src="img/bitcoin.png"
+        class="commons_logo"
+        alt="bitcoin Logo"
+      />
+    </section>
+
+    <section class="events" id="events">
+      <h2>Discover more info on Green energy hot topics</h2>
+      <hr />
+      <p>
+        <ul>
+          <li>Make renewable energy technology a global public good</li>
+          <li>Improve global access to components and raw materials</li>
+          <li>Level the playing field for renewable energy technologies</li>
+          <li>Triple investments in renewables</li>
+          <li>Triple investments in renewables</li>
+        </ul>
+      </p>
+      <div class="event_list">
+        <div class="event_container">
+          <div class="orange">
```

Comment:
- [ ] great job. please ensure both images are of same shape and size, also centralized so it matches the original design
yours
![Screenshot (611)_LI](https://user-images.githubusercontent.com/65298048/203499897-bd7f7128-14d0-4447-bff9-bd29aff4aef4.jpg)

original design
![Screenshot (614)](https://user-images.githubusercontent.com/65298048/203499999-59f091f9-7d84-4f35-b80b-dd6388d5b21a.png)